### PR TITLE
Feature/api 2

### PR DESCRIPTION
**Pull request by wijohnst**
Added Jest as a testing framework and wrote a simple "Hello world" test.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,51 +14,44 @@
```
Comment (vijayyadav06) [inline README.md]: super thanks for updating markdown!

**Comment by mollyhenderson** [comment]
Hey Will! Thanks for adding this!

We don't actually want to merge all the `api-2` work into the `master` branch until all the other cards are completed... so this PR **shouldn't be merged** for now. The testing framework is in the feature branch though, which is really what we needed to start adding tests :)

**Comment by wijohnst** [comment]
Molly,

**Comment by wijohnst** [comment]
Molly,

As always, thank you for being so patient with me as I figure out this whole 'working in a group' thing! It's nice to be able to practice something like this in a safe environment; I really appreciate it. Now to write some tests!

**Pull request closed without merging**